src/main/java/de/samply/share/broker/rest/Monitoring.java

```diff
@@ -221,18 +222,18 @@ public Response createQuery(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
       @Parameter(
           name = "query",
           description = "Query as a JSON object",
-          schema = @Schema(implementation = QueryObject.class))
-          QueryObject queryObject) {
+          schema = @Schema(implementation = QueryContainer.class))
+          QueryContainer<CqlQuery[]> queryContainer) {
```

Comment:
I'm sure we sad, that this endpoint should be kept until the monitoring solution is changed. So please introduce a new endpoint for the QueryContainer and remove this endpoint after the monitoring is changed.